package/src/utils/define-integration.ts

```diff
@@ -69,8 +69,13 @@ export const defineIntegration = <
 						}),
 					addVitePlugin: (plugin) =>
 						addVitePlugin({ plugin, updateConfig: params.updateConfig }),
-					hasIntegration: (name) =>
-						hasIntegration({ name, config: params.config }),
+					hasIntegration: (other: string, position?: "before" | "after", relativeTo?: string) =>
+						hasIntegration({
+							name: other,
+							relativeTo: relativeTo ?? name,
```

Comment:
Is there a reason for setting a fallback? Examples to help me understand what's going on here would be helpful ngl 😅

Reply:
If you call `hasIntegration("@astrojs/mdx", "before")` the `relativeTo` will be undefined, the fallback makes it is relative the current integration.

I'll add a comment there